recipes/ragel/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ragel" %}
+{% set version = "6.6" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "a8f38166d57163ff821ad4608ba258ed3b01ac8abb890440e03163cbb835e932" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{version }}.tar.gz
+  url: https://www.colm.net/files/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    # Patch shamelessly taken from MacPorts.
+    - patches/cxx_syntax.patch
+
+build:
+  number: 0
+  # TODO: Try to build on windows.
```

Comment:
Please remove the comment and create an issue in the feedstock once that is created. (Comments are usually lost, issue can be tracked.)